### ISSUE TITLE
llvm build error with boost 1.62.0

### ISSUE THREAD

**Issue by kyleterry** [tekezo/Karabiner-Elements#187]
Git revision: ab78f8a69a68ad7a9f6b0475e6cab3afca41677b
Xcode: 8.0
Xcode CLI tools: installed
Boost: 1.62.0
macOS: 10.12 (16A323)

```
The following build commands failed:
	CompileC build/karabiner_console_user_server.build/Release/karabiner_console_user_server.build/Objects-normal/x86_64/main.o main.cpp normal x86_64 c++ com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
make[3]: *** [all] Error 65
make[2]: *** [all] Error 2
make[1]: *** [build] Error 2

make: *** [all] Error 99
```

**Comment by hmps** [comment]
I had this issue too and it seems to be a problem with Boost 1.6.2. After downgrading to 1.61.0 my build is running without a hitch.

**Comment by kyleterry** [comment]
@hmps Thanks! That was indeed it. I changed the name of the issue to include the boost version so this can be addressed. The readme says 1.61.0+ so in theory that should include 1.62.0.

**Comment by tekezo** [comment]
Thank you for feedback!
I've fixed this issue at f5f5a37262490bf4f78d9f3f9ccaea49e3581445.
